src/vnc.yml

```diff
@@ -1,31 +1,101 @@
 ---
 - hosts: all
-  name: Install VNC
+  name: Install VNC and configure vnc user
```

Comment:
`vnc user` and `VNC user` are both being used. I would like us to pick one and use it consistently.

Reply:
I prefer `VNC user`, since it the username is configurable.

Reply:
See commit 06e0f17.